File: ds/fibi.c
#include<stdio.h>
int fibi(int n,int t1, int t2){
	if(n>0){
		printf("%d ",t1);
		fibi(n-1,t2,(t1+t2));
	}else 
		return 0;
}
void main(){
	int n;
	printf("Enter Number of Series to be Generated:");
	scanf("%d",&n);
	printf("%d Fibinocci Numbers are;\n",n);
	fibi(n,0,1);
	printf("\n");
}
//	t3 = t1 + t2;
//	t1 = t2;
//	t2 = t3
//	t1 = 0
//	t2 = 1
//	t3 = 0 + 1
//	
//	t1 = 1 [t2]
//	t2 = 1 [t3]

File: php/fibi.php
<!DOCTYPE html> 
<html>
   <head>
      <title></title>
   </head>
   <body>
      <?php if(!$_POST){     ?>         
      <form method="POST">
         <input type="number" name="n1">                      
         <input type="submit"> 
      </form>
      <?     
      }else{         
      function fibi($a, $b, $n) { 
         static $count; 
         if($n==$count)                 
            return; 
         else{                 
            echo ' '.$a;
            $count++;
            fibi($b,$a+$b,$n); 
         }         
      }
         $x = $_POST['n1'];
         echo "<h3>$x Fibinocci Numbers Are;</h3><strong>";
   		fibi(0,1,$x);
   	}
   	?>
   </body>
</html>